kolibri_zim_plugin/assets/src/views/ZimBreadcrumbsMenu.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+
+  <transition-group class="breadcrumbs-list" name="breadcrumbs-fade" mode="out-in" tag="ol">
+    <template v-for="(breadcrumb, index) in visibleBreadcrumbs">
+      <li
+        :ref="`breadcrumb${index}`"
+        :key="index"
+        class="breadcrumb-item"
+      >
+        <KButton
```

Comment:
I guess it makes sense to use Kolibri UI components and the kolibri-design-system in this content plugin.

Reply:
Indeed, the idea here is to be consistent with the other renderer plugins to keep things upstreamable :)